apps/dash-simple-image-processor/requirements.txt

```diff
@@ -0,0 +1,4 @@
+black==19.10b0
+dash-canvas==0.1.0
```

Comment:
is dash-canvas used anywhere in this demo? I also don't think black is required here, not sure about wheel.

Reply:
you need scikit-image though for your `io_utils.py`, and scikit-image is a dependency of `dash-canvas`. But if `dash-canvas` is not needed per se it's better to just put scikit-image in the requirements.

Reply:
This is the output of `pipdeptree | grep -P '^\w+'` which programmatically determines the minimum `requirements.txt` file. But it's true `black` is probably not needed.

Reply:
Basically `dash-canvas` requires everything we need haha.

Reply:
`scikit-image` is installed as a dependency of `dash-canvas`
```
dash-canvas==0.1.0
  - dash [required: >=1.6.1, installed: 1.12.0]
    - dash-core-components [required: ==1.10.0, installed: 1.10.0]
    - dash-html-components [required: ==1.0.3, installed: 1.0.3]
    - dash-renderer [required: ==1.4.1, installed: 1.4.1]
    - dash-table [required: ==4.7.0, installed: 4.7.0]
    - Flask [required: >=1.0.2, installed: 1.1.2]
      - click [required: >=5.1, installed: 7.1.2]
      - itsdangerous [required: >=0.24, installed: 1.1.0]
      - Jinja2 [required: >=2.10.1, installed: 2.11.2]
        - MarkupSafe [required: >=0.23, installed: 1.1.1]
      - Werkzeug [required: >=0.15, installed: 1.0.1]
    - flask-compress [required: Any, installed: 1.5.0]
      - brotli [required: Any, installed: 1.0.7]
      - flask [required: Any, installed: 1.1.2]
        - click [required: >=5.1, installed: 7.1.2]
        - itsdangerous [required: >=0.24, installed: 1.1.0]
        - Jinja2 [required: >=2.10.1, installed: 2.11.2]
          - MarkupSafe [required: >=0.23, installed: 1.1.1]
        - Werkzeug [required: >=0.15, installed: 1.0.1]
    - future [required: Any, installed: 0.18.2]
    - plotly [required: Any, installed: 4.8.1]
      - retrying [required: >=1.3.3, installed: 1.3.3]
        - six [required: >=1.7.0, installed: 1.15.0]
      - six [required: Any, installed: 1.15.0]
  - Pillow [required: Any, installed: 7.1.2]
  - scikit-image [required: Any, installed: 0.17.2]
```

Reply:
Basically what I'm trying to get at is that  the best looking `requirements.txt` is not really well defined ;). But I understand we're supposed to showcase `dash` and `scikit-image`, so what if I just include those 2 as well, so people that look won't be confused (it shouldn't break anything).